documentation/markup.rst

```diff
@@ -1019,13 +1019,20 @@ units as well as normal text:
    feature, or a part of it, to the library or C API.  When this applies to an
    entire module, it should be placed at the top of the module section before
    any prose.
+   When a new attribute, function, or method is added, a
+   ``versionadded`` directive should be present at the end of its
+   documentation block.
```

Comment:
**Note**: _Requires adding a new ref target label, `.. _information-units`, above the eponymous section above.

```suggestion
   When adding a new API :ref:`with a directive <information-units>`
   (``class``, ``attribute``, ``function``, ``method``, ``c:type``, etc),
   a ``versionadded`` should be included at the end of its description block.
```

* More broadly construe the types of entities we're discussing here (not just attributes, functions and methods)
* Be more specific about which get `versionadded` (any object/API that is represented by a directive, vs. e.g. callable parameters, behaviors, etc. that are not).
* Crosslink the relevant section
* Use active voice
* Describe the location a little more precisely/less ambiguously